base/broadcast.jl

```diff
@@ -283,6 +288,15 @@ function broadcast_c(f, ::Type{Tuple}, As...)
     n = length(shape[1])
     return ntuple(k->f((_broadcast_getindex(A, k) for A in As)...), n)
 end
+function broadcast_c(f, ::Type{Nullable}, a...)
+    nullity = any(isnull, a)
```

Comment:
Yes, that's what I was planning to do. I will have a look at that PR and potential ways to reduce allocation.